src/main/webapp/dev/mashup.jsp

```diff
@@ -20,7 +20,11 @@
         <script src="/js/common.js" type="text/javascript"></script>
     </head>
     <body>
-        <div class="container theme-showcase">
+    <h1 style="text-align: center">This page contains a combination
+        of all pages of the TEAMMATES app. It is used for manually
+        inspecting the TEAMMATES UI for visual defects. To load this
+        page, you need to run the test MashupPageUiTest.java.</h1>
+    <div class="container theme-showcase">
```

Comment:
I have removed the files that are associated with this issues , please revise this and let me know if there is any change that i need to revert